matching_brackets.exs

```diff
@@ -0,0 +1,49 @@
+defmodule BracketsBalance do
+  @brackets %{
+    "{" => "}",
+    "(" => ")",
+    "[" => "]"
+  }
+
+  def add_to_stack(stack, counter, check_string) do
+    if counter < length(check_string) do
+      stack =
+        if Map.has_key?(@brackets, Enum.at(check_string, counter)) do
+          stack ++ [Enum.at(check_string, counter)]
+        else
+            if Enum.at(check_string, counter) == @brackets[Enum.at(stack, -1)] do
+                stack |> Enum.reverse() |> tl() |> Enum.reverse()
```

Comment:
And here you don't need to double-reverse, you just need to remove first element:
```
tl(stack)
```